scalableminds/webknossos
# Add more information to task csv download

### URL of deployed dev instance (used for testing):
- https://verbosetaskcsvdownload.webknossos.xyz

### Steps to test:
- open the task view and download some tasks
- the first two entries are the task id and the creation time -> remove them from each line
- open the task bulk creation and paste in the left-over content of the csv
- try creating those tasks
- each step should work 

### Issues:
- fixes #4642
------
